src/vs/workbench/contrib/welcome/page/browser/az_data_welcome_page.ts

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
can we add this file under `sql` directory?  New files should be created in parallel directories under there.